hack/generate-manifest.sh

```diff
@@ -201,8 +201,8 @@ if [ "$TUN_TYPE" != "geneve" ] && [ "$TUN_TYPE" != "vxlan" ] && [ "$TUN_TYPE" !=
     exit 1
 fi
 
-if [ "$MODE" == "release" ] && [ -z "$IMG_NAME" ]; then
-    echoerr "In 'release' mode, environment variable IMG_NAME must be set"
+if [ "$MODE" == "release" ] && [ -z "$AGENT_IMG_NAME" ] || [ -z "$CONTROLLER_IMG_NAME" ]; then
```

Comment:
In release mode, if `AGENT_IMG_NAME` is set and CONTROLLER_IMG_NAME is unset, the condition result is still false.